monai/apps/pathology/transforms/post/dictionary.py

```diff
@@ -286,44 +264,38 @@ class GenerateWatershedMarkersd(MapTransform):
 
     def __init__(
         self,
-        keys: KeysCollection,
+        mask_key: str = "mask",
         border_key: str = "border",
         markers_key: str = "markers",
         threshold: float = 0.4,
         radius: int = 2,
-        min_size: int = 10,
-        remove_small_objects: bool = True,
+        min_object_size: int = 10,
         postprocess_fn: Optional[Callable] = None,
         dtype: DtypeLike = np.uint8,
-        allow_missing_keys: bool = False,
     ) -> None:
-        super().__init__(keys, allow_missing_keys)
+        self.mask_key = mask_key
         self.border_key = border_key
         self.markers_key = markers_key
```

Comment:
Do we need to check whether these keys are empty strings? They obviously have defaults set, but if passed as "" I guess that could cause problems. This may be overkill, so feel free to ignore!

Reply:
Actually empty string is a valid key in python! ;)